doc/source/commandline.rst

```diff
@@ -28,6 +28,8 @@ options.
    `here <http://dynet.readthedocs.io/en/latest/minibatch.html>`_.
 -  ``--dynet-gpus NUMBER``: Specify how many GPUs you want to use, if
    DyNet is compiled with CUDA. Currently, only one GPU is supported.
+-  ``--dynet-gpu``: Specify whether to use GPU or not. Note that it is the only option
```

Comment:
Do you mean "this is only an option for Python programs"? Is there a reason why we can't support it for C++ also?

Reply:
I've no idea actually. The old version of DyNet only has this option in the Python layer. I just follow the previous rule(https://github.com/clab/dynet/blob/9606e2a3fdc31788b3739adfcfb498b1626645fd/python/dynet.py#L5).